client/src/components/Home.tsx

```diff
@@ -0,0 +1,14 @@
+import {RaisedButton} from 'material-ui';
+import Page from './util/Page';
+import * as React from 'react';
+import { browserHistory } from 'react-router';
+
+
+export const Home = (_: any) => {
+    return (<Page>
+        <RaisedButton primary={true} label='New Lesson'
```

Comment:
Cuz when we hookup to backend it'll need to be more complex and I wanted a button